src/scss/theme/_tables.scss

```diff
@@ -0,0 +1,82 @@
+@charset "UTF-8";
+
+// Special table for VPUE.
+.resp_table { 
+    width: 100%;
```

Comment:
Scss should be indented with 2 spaces, not 4.

Reply:
@pookmish Oops. I'll fix.